src/services/refactors/moveToFile.ts

```diff
@@ -179,7 +179,8 @@ registerRefactor(refactorNameForMoveToFile, {
             }
         }
         if (context.preferences.allowTextChangesInNewFiles && statements) {
-            return [{ name: refactorNameForMoveToFile, description, actions: [moveToFileAction] }];
+            const affectedTextRange = { start: statements.all[0].pos, length: statements.all.length > 1 ? last(statements.all).end - statements.all[0].pos : statements.all[0].end - statements.all[0].pos };

```

Comment:
Same here with use of `.pos`; it might make more sense to use `.getStart(sourceFile)`